build/configs/artik053s/st_things/defconfig

```diff
@@ -248,7 +248,7 @@ CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_MOUNTPOINT="/sss"
 CONFIG_ARTIK05X_AUTOMOUNT_ROMFS=y
 CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_DEVNAME="/dev/mtdblock9"
 CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_MOUNTPOINT="/rom"
-CONFIG_SVR_DB_SECURESTORAGE=y
```

Comment:
I agree with you in ARTIK053.
However, unlike ARTIK053, ARTIK053s has build-in secure storage, intending to save information in a safe way.
So, it might be better to have the secure storage config option by default.

Reply:
I agree.
I will modify default config of 053s.
H/W KEY CERT FEATURE is good to get dependency to SECURE STORAGE only for 053s.